src/RedisResourceManager.php

```diff
@@ -123,6 +123,22 @@ public function getPassword($id)
         return $resource['password'];
     }
 
+    /**
+     * Get redis resource user
+     *
+     * @param string $id
+     * @return string
+     */
+    public function getUser($id)
```

Comment:
Since this is a new method, we can add argument type here.
```suggestion
    public function getUser(string $id): string
```